dotcom-rendering/src/web/components/KeyEventsCards.tsx

```diff
@@ -0,0 +1,110 @@
+import { css } from '@emotion/react';
+import { textSans, neutral, space, from } from '@guardian/source-foundations';
+import { Link } from '@guardian/source-react-components';
+import { timeAgo } from '@guardian/libs';
+import { decidePalette } from '../lib/decidePalette';
+
+export interface KeyEvent {
```

Comment:
This is a different model than what a key event is - they are just of type Block. I wonder if we should pass in key events as blocks here and move the logic that determines what's a date, text, url etc into this component?